src/main/kotlin/io.javalin/translator/template/Freemarker.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2017 David Åse
+ * Licensed under Apache 2.0: https://github.com/tipsy/javalin/blob/master/LICENSE
+ */
+
+package io.javalin.translator.template
+
+import freemarker.template.Configuration
+import freemarker.template.TemplateException
+import freemarker.template.Version
+import java.io.IOException
+import java.io.StringWriter
+
+object Freemarker {
```

Comment:
Kotlin objects can inherit interfaces so all those renderers could inherit a common interface and maybe have some shared behaviour or some pattern when they are used